File: python_programming_basics/conditional_statements_advanced/e_09_ski_trip.py
days = int(input())
place = input()
grade = input()
price = 0

if place == "room for one person":
    price = (days - 1) * 18

elif place == "apartment":
    price = (days - 1) * 25

    if days < 10:
        price -= price * 0.3

    elif 10 <= days <= 15:
        price -= price * 0.35

    elif days > 15:
        price -= price * 0.5

elif place == "president apartment":
    price = (days - 1) * 35

    if days < 10:
        price -= price * 0.1

    elif 10 <= days <= 15:
        price -= price * 0.15

    elif days > 15:
        price -= price * 0.2

if grade == "positive":
    price += price * 0.25

else:
    price -= price * 0.1

print(f"{price:.2f}")

File: JavaScript Basics/03. Conditional Statements Advanced/e-09-Ski-Trip.js
function calculateSkiTripCost(days, roomType, feedback) {
    let nights = days - 1;
    let pricePerNight = 0;
    let discount = 0;

    if (roomType === "room for one person") {
        pricePerNight = 18.00;
    } else if (roomType === "apartment") {
        pricePerNight = 25.00;
        if (nights < 10) {
            discount = 0.30;
        } else if (nights <= 15) {
            discount = 0.35;
        } else {
            discount = 0.50;
        }
    } else if (roomType === "president apartment") {
        pricePerNight = 35.00;
        if (nights < 10) {
            discount = 0.10;
        } else if (nights <= 15) {
            discount = 0.15;
        } else {
            discount = 0.20;
        }
    }

    let totalCost = nights * pricePerNight * (1 - discount);

    if (feedback === "positive") {
        totalCost *= 1.25;
    } else if (feedback === "negative") {
        totalCost *= 0.90;
    }

    console.log(totalCost.toFixed(2));
}

calculateSkiTripCost(14, "apartment", "positive");
calculateSkiTripCost(30, "president apartment", "negative");
calculateSkiTripCost(12, "room for one person", "positive");
calculateSkiTripCost(2, "apartment", "positive");
